static/src/js/actions/preferences.js

```diff
@@ -50,6 +52,12 @@ export const updatePreferences = data => (dispatch, getState) => {
         resource: 'preferences',
         requestObject
       }))
+
+      if (error.message === 'Network Error') {
```

Comment:
When this event fires, and an error occurs, we see the error banner and the toast -- I think it'd be a good change to just add a flag to `addError` for `notificationType` that could be one of `[banner, toast, none]` and call this `pushErrorNotification` if the value is toast.

Reply:
Will do.